src/FoundMovieCounter/FoundMovieCounter.tsx

```diff
@@ -1,24 +1,18 @@
-import PropTypes from 'prop-types';
-
-import './style.css';
-
-function FoundMovieCounter({ number }) {
-  return (
-    <div className="results">
-      <div className="movie-counter">
-        <strong>
-          {number}
-        </strong>
-
-        {' '}
-        movies found
-      </div>
-    </div>
-  );
-}
-
-FoundMovieCounter.propTypes = {
-  number: PropTypes.number.isRequired,
-};
-
-export default FoundMovieCounter;
+import './style.css';
+
+export default function FoundMovieCounter(
+  { number }: { number: number },
```

Comment:
number prop should be renamed to the amount, and the interface should be extracted.

Reply:
Renamed and extracted.